test/libsolidity/ASTJSON/long_type_name_binary_operation.json

```diff
@@ -8,6 +8,7 @@
     ]
   },
   "id": 12,
+  "license": null,
```

Comment:
Didn't we had some PR/issue discussing that we shouldn't have `null` fields? Maybe it is out of scope here.

Reply:
there is an open pr about it.